<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>361</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>453</width>
    <height>241</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_8" >
   <item row="0" column="0" >
    <layout class="QGridLayout" name="gridLayout_7" >
     <property name="sizeConstraint" >
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item row="0" column="0" >
      <layout class="QGridLayout" name="gridLayout_4" >
       <item row="1" column="0" >
        <layout class="QGridLayout" name="gridLayout_3" >
         <item row="0" column="0" >
          <widget class="QCheckBox" name="cbCreateServer" >
           <property name="text" >
            <string>Create server</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QSpinBox" name="sbClientsCount" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="minimum" >
            <number>3</number>
           </property>
           <property name="maximum" >
            <number>4</number>
           </property>
           <property name="value" >
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="0" >
        <layout class="QGridLayout" name="gridLayout" >
         <item row="1" column="0" >
          <widget class="QLineEdit" name="leServerIP" >
           <property name="text" >
            <string>localhost</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QListWidget" name="lwServersList" />
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="1" >
      <layout class="QGridLayout" name="gridLayout_6" >
       <item row="0" column="0" >
        <layout class="QGridLayout" name="gridLayout_5" >
         <item row="0" column="0" >
          <widget class="QPushButton" name="pbSearch" >
           <property name="text" >
            <string>Start search</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QPushButton" name="pbConnect" >
           <property name="text" >
            <string>Connect to server</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" >
        <layout class="QGridLayout" name="gridLayout_2" >
         <item row="0" column="0" >
          <widget class="QLabel" name="lPort" >
           <property name="text" >
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="lNickname" >
           <property name="text" >
            <string>Nickname</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLineEdit" name="leNickname" />
         </item>
         <item row="1" column="0" >
          <widget class="QSpinBox" name="sbPort" >
           <property name="minimum" >
            <number>1024</number>
           </property>
           <property name="maximum" >
            <number>65535</number>
           </property>
           <property name="value" >
            <number>1500</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" >
        <widget class="QComboBox" name="comboBox" >
         <item>
          <property name="text" >
           <string>Красный</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Желтый</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Зеленый</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Синий</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QTextEdit" name="textEdit" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbCreateServer</sender>
   <signal>toggled(bool)</signal>
   <receiver>leServerIP</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>65</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel" >
     <x>135</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbCreateServer</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbClientsCount</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>65</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel" >
     <x>56</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbCreateServer</sender>
   <signal>toggled(bool)</signal>
   <receiver>lwServersList</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>66</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel" >
     <x>153</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lwServersList</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>leServerIP</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>153</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel" >
     <x>153</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
